packages/datadog_flutter_plugin/doc/rum/error_tracking.md

```diff
@@ -31,61 +27,41 @@ final configuration = DdSdkConfiguration(
 
-Once your application crashes and you restart the application, the Datadog iOS SDK uploads a crash report to Datadog. 
 
-You can use the [@datadog/datadog-ci][4] command line tool to upload your dSYM file. By default, Flutter adds these dSYM files in `./build/ios/archive/Runner.xcarchive/dSYMs`. After building your application with `flutter build ipa`, run the following shell command to upload your dSYMs to Datadog:
+The [@datadog/datadog-ci][4] command line tool supports uploading all of the necessary files (dSYMs, Android Proguard Mapping, and Dart Symbol Files) in one command.
 
+First, install the `datadog-ci` tool from the instructions above and create a `datadog-ci.json` file at the root of your project, containing your API key and (optionally) your Datadog site:
 ```sh
-export DATADOG_API_KEY="<API KEY>"
-
-npx @datadog/datadog-ci dsyms upload ./build/ios/archive/Runner.xcarchive/dSYMs
-```
-
-To configure the tool using an EU endpoint, set the `DATADOG_SITE` environment variable to `datadoghq.eu`. To override the full URL for the intake endpoint, define the `DATADOG_DSYM_INTAKE_URL` environment variable. 
-
-## Manually upload Android ProGuard mapping files to Datadog
-
-If you are using the `--obfuscate` parameter on Android builds, and you wish to deobfuscate your traces, you need to upload your ProGuard mapping file to Datadog. The [Datadog Android SDK Gradle plugin][5] supports uploading your mapping file directly to Datadog.
-
-You can configure the plugin by adding the following lines to your `./android/app/build.gradle` file:
-
-```
-plugins {
-    id("com.datadoghq.dd-sdk-android-gradle-plugin") version "x.y.z"
+{
+  "apiKey": "<YOUR_DATADOG_API_KEY>",
+  "datadogSite": "datadoghq.eu"  // Optional if you are using datadoghq.com
 }
 ```
 
-Additionally, if you need to configure the upload, add the following to the end of your `./android/app/build.gradle` file:
+Because this file contains your API key, it should not be checked in to version control.
 
-```
-datadog {
-    // versionName is optional, by default it is read from your Android plugin configuration's version name,
-    // which is set from you pubspec.yaml at build time
-    versionName = "1.3.0" 
-    serviceName = "my-service" // Optional, by default it is read from your Android plugin configuration's package name
-    site = "US" // Optional, can be "US", "EU" or "GOV". Default is "US"
-}
+Alternately, you can set the `DATADOG_API_KEY` and `DATADOG_SITE` environment variables. 
+
+Then, you can use the following command to upload all the necessary files for symbolication and deobfuscation of your crash reports:
+```sh
+datadog-ci flutter-symbols upload --service-name <your_service_name> --dart-symbols-location <location_of_dart_symbols> --android-mapping --ios-dsyms
```

Comment:
why dSyms are mentioned in both `datadog-ci flutter-symbols` call and `npx @datadog/datadog-ci dsyms upload` call?

Reply:
Looks like there's some lines tat the end of the readme that shouldn't be there. There shouldn't be any mention of `dsyms upload`